includes/Modules/Analytics.php

```diff
@@ -348,6 +349,10 @@ protected function get_datapoint_definitions() {
 	 */
 	protected function create_data_request( Data_Request $data ) {
 		switch ( "{$data->method}:{$data->datapoint}" ) {
+			case 'GET:account-summaries':
+				return function() {
+					return $this->get_service( 'analytics' )->management_accountSummaries->listManagementAccountSummaries();
+				};
```

Comment:
This must not be wrapped in a closure. Wrapping in a function should only be used if the data point logic relies primarily on custom code. If it is for a Google API request (like here), the API request itself needs to be returned (see e.g. `POST:create-account-ticket` below). This allows the batch mechanism to work correctly and combine these requests, and the post-processing then happens in `parse_data_response`.